### Kuon kuon_control/BrakeCmd

# Motor braking is either rheostatic, if the generated electrical power during
# braking is dissipated as heat, and regenerative (future Kuon designs).

float32 val      # motor dynamic braking intensity (RDK: move to 0 - 1 scale)
